packages/legend-application-studio/src/stores/StoreRelational_LegendStudioApplicationPlugin_Extension.ts

```diff
@@ -97,4 +114,29 @@ export interface StoreRelational_LegendStudioApplicationPlugin_Extension
    * Get the list of renderers for the editor for a relational database authentication strategy.
    */
   getExtraAuthenticationStrategyEditorRenderers?(): AuthenticationStrategyEditorRenderer[];
+
+  // --------------------- relational database connection post processor ------------------
+
+  /**
+   * Get the list of the supported type specifiers for relational post processors
+   */
+  getExtraPostProcessorTypes?(): string[];
+
+  /**
+   * Get the list of classifiers for a relational post processor specification.
+   */
+  getExtraPostProcessorTypeGetters?(): PostProcessorTypeGetter[];
+
+  /**
+   * Get the list of creators for relational post processor specification given the type specifier.
+   */
+  getExtraPostProcessorCreators?(
```

Comment:
This looks weird, why would you need to provide some params to even get this list?